docs/sources/reference/components/prometheus/prometheus.exporter.windows.md

```diff
@@ -198,9 +198,9 @@ The v2 collector can query service states much more efficiently, but can't provi
 
 ### smb block
 
-Name           | Type     | Description                                | Default | Required
----------------|----------|--------------------------------------------|---------|---------
-`enabled_list` | `string` | Comma-separated list of collectors to use. | `""`    | no
+Name           | Type     | Description                                                    | Default | Required
+---------------|----------|----------------------------------------------------------------|---------|---------
+`enabled_list` | `string` | Deprecated (no-op), Comma-separated list of collectors to use. | `""`    | no
```

Comment:
Is there a replacement for this?

Reply:
The initial author of that collector implement the same sub-collector concept, as we have for exchange collector. 

However there was only one collector (for smb and smb_client). The new behavior is just to enable the single sub-collector by default.

Reply:
Cool, can we add that to the doc and/or changelog?

Reply:
I add something to the Changelog, could you check if sufficient?